### PR TITLE
Package ocplib-simplex.0.4.1

### PR DESCRIPTION
**Pull request by Halbaroth**
### `ocplib-simplex.0.4.1`
A library implementing a simplex algorithm, in a functional style, for solving systems of linear inequalities
This library is a (fully) functional OCaml implementation of the simplex algorithm for solving systems of linear inequalities. The implementation is incremental and backtrackable. It is able to extract unsat-cores for unsatisfiable problems. Versions > 0.1 also support linear optimization.

---
* Homepage: https://github.com/OCamlPro-Iguernlala/ocplib-simplex
* Source repo: git+https://github.com/OCamlPro-Iguernlala/ocplib-simplex.git
* Bug tracker: https://github.com/OCamlPro-Iguernlala/ocplib-simplex/issues

---
:camel: Pull-request generated by opam-publish v2.2.0

### CODE REVIEW

File: packages/ocplib-simplex/ocplib-simplex.0.4.1/opam [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  "num"
+]
+flags: light-uninstall
+build: [
+  ["autoconf"]
+  ["./configure" "-prefix" "%{prefix}%"]
+  [make]
+]
+install: [make "install"]
+remove: ["ocamlfind" "remove" "ocplib-simplex"]
```
Comment (kit-ty-kate) [inline packages/ocplib-simplex/ocplib-simplex.0.4.1/opam:29]:
```suggestion
```

File: packages/ocplib-simplex/ocplib-simplex.0.4.1/opam [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+license: "LGPL-2.1-or-later"
+homepage: "https://github.com/OCamlPro-Iguernlala/ocplib-simplex"
+bug-reports: "https://github.com/OCamlPro-Iguernlala/ocplib-simplex/issues"
+depends: [
+  "ocaml" {>= "4.01.0"}
+  "ocamlfind" {build}
+  "conf-autoconf" {build}
+  "num"
+]
+flags: light-uninstall
```
Comment (kit-ty-kate) [inline packages/ocplib-simplex/ocplib-simplex.0.4.1/opam:22]:
```suggestion
```

**Review by kit-ty-kate — COMMENTED** [review]
These are not necessary since opam 2.0

**Comment by Halbaroth** [comment]
Thanks :)

**Comment by kit-ty-kate** [comment]
Thanks

**Pull request merged**